pylint/checkers/utils.py

```diff
@@ -876,7 +876,11 @@ def uninferable_final_decorators(
             except AttributeError:
                 continue
         elif isinstance(decorator, nodes.Name):
-            import_node = decorator.lookup(decorator.name)[1][0]
+            lookup_values = decorator.lookup(decorator.name)
+            if lookup_values[1]:
+                import_node = decorator.lookup(decorator.name)[1][0]
+            else:
+                continue  # pragma: no cover # Covered on Python < 3.8
```

Comment:
```suggestion
            try:
                 import_node = decorator.lookup(decorator.name)[1][0]
             except IndexError:
                 continue
```

Reply:
I just committed your other suggestion. Do you prefer the `try ... except`?

Reply:
No I don't feel strongly about it; I would keep it as you have it unless you have a reason to change it.